networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -528,6 +530,18 @@ def get_connected_devices(self, switches: List[NbRecord]) -> Tuple[Set[NbRecord]
                 # Those that we find unbundled are probably not modelled correctly, so we exclude them.
                 if not iface.lag:
                     continue
+                m = self.lag_id_re.match(iface.lag.name)
+                if not m:
+                    print(f'Cannot parse LAG id on interface {switch.name}/{iface.name}, '
+                          f'bundled in {iface.lag.name}')
+                lag_id = int(m.group('lag_id'))
+
+                if not RESPONSIBLE_LAG_RANGES[0] < lag_id < RESPONSIBLE_LAG_RANGES[1]:
+                    if self.verbose:
+                        print(f'LAG id {lag_id} on {switch.name}/{iface.lag.name} is oustide of driver '
+                              f'responsible bounds [{RESPONSIBLE_LAG_RANGES[0]}, {RESPONSIBLE_LAG_RANGES[1]}]')
```

Comment:
Mathematically this would also hint for an inclusive range, as `[a, b]` is generally used to denote that in math. `(a, b)` would be exkclusive.

https://en.wikipedia.org/wiki/Interval_(mathematics)#Including_or_excluding_endpoints

Reply:
Fair point.